src/lib/config.ts

```diff
@@ -536,6 +536,13 @@ export class default_config {
         hintmaps: "false",
     }
 
+    /**
+     * Instead of fetching actual character which depends on layout,
+     * use machine code of a key.
+     * Recommended for non-latin or non-qwerty users.
+     */
+    usecodesinsteadofkeys: "true" | "false" = "true"
```

Comment:
We should work hard to make this setting easy to find - probably mention internationalization, Dvorak, bépo, Russian in the title so it is easy to find.

In theory I'm happy to make it be the default for new users but we should use a config updater (and make sure it works!) to make sure it's false for current users. Otherwise we will break Tridactyl for ~hundreds of people overnight.

Reply:
How can we distinguish between an update and new installation?

Reply:
In theory new installations of Tridactyl will have `:set configversion 0.0` - https://github.com/tridactyl/tridactyl/blob/65586aa61c549f83f3d2c335cd1ecd12d093f392/src/lib/config.ts#L77 - and existing installations will have a higher number.

So we should be able to add a line here https://github.com/tridactyl/tridactyl/blob/65586aa61c549f83f3d2c335cd1ecd12d093f392/src/lib/config.ts#L1591 that turns on the keycode mode.

Really though I think we should make it so the default config version is whatever the current config version is, then we could instead have an updater that set the keycode mode for the legacy installations and have the real default be easily found in the configuration. I'm not sure why it doesn't work like that currently.

I'm not very sure that the config updater works reliably - maybe we could replace https://github.com/tridactyl/tridactyl/blob/65586aa61c549f83f3d2c335cd1ecd12d093f392/src/background.ts#L287 with a `config.getAsync(...).then`